.changeset/lazy-ducks-complain.md

```diff
@@ -0,0 +1,46 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+
+- ButtonIcon
+
+---
+
+Add optional `tooltipPlacement` prop to `ButtonIcon`
+
+The `tooltipPlacement` prop allows you to specify the placement of the tooltip to either `top` or `bottom`. The
+default value is `top`.
```

Comment:
```suggestion
The `tooltipPlacement` prop allows you to specify the placement of the tooltip to either `top` or `bottom`.
The default value is `top`.
```

We usually keep one sentence per line to avoid noisy diffs and help with readability.